src/gravatar/web_api.js

```diff
@@ -1,8 +1,10 @@
-import { md5 } from 'blueimp-md5';
+import blueimp from 'blueimp-md5';
 import { validateEmail } from '../cred/index';
 import jsonp from '../utils/jsonp_utils';
 import * as preload from '../preload/index';
 
+const md5 = blueimp.md5 || blueimp;
```

Comment:
Yes, although the [approach](https://github.com/auth0/lock/blob/b0a587953370d746a9a446bf6730c824d3958d65/lib/md5/index.js#L3) taken in lock is cleaner I didn't want to introduce another module.

Also tried replacing it with the `js-md5` package. It worked but added a lot of kB to the build.